### PR TITLE
feat: modify wa-sqlite web app to use DAL and demonstrate migrations

### PR DESCRIPTION
**Pull request by kevin-dp**

### CODE REVIEW

**Comment by thruflo** [comment]
Great to have as reference 👍

One thing to coordinate with @balegas — I think we want to keep the examples repo main branch working with the published npm client. Thus I believe this should be in the monorepo examples folder and we shouldn’t merge changes into main here that use the new version as a dependency and diverge from the website documentation for now.

File: web-wa-sqlite/README.md [hunk 1]
```diff
@@ -127,6 +121,73 @@ const ExampleComponent = () => {
...
+docker exec -it -e PGPASSWORD=password local-stack-postgres_1-1  psql -h 127.0.0.1 -U postgres -d electric
+electric=# ALTER TABLE items ADD other_value TEXT;
+WARNING:  assigning automatic migration version id: 20230529120539_974
+ALTER TABLE
+```
+
+This migration will automatically be picked up by Electric and will be streamed to the application
+which will migrate its local SQLite database.
+
+Then, remains to migrate the code of our application to do something with the new column.
```
Comment (balegas) [inline web-wa-sqlite/README.md:139]: I think we can be more precise here. 
Is migrating the code the best way of saying it? Then, I think we could say that what we actually are doing is to migrate the type model to know about the new column.

File: web-wa-sqlite/README.md [hunk 2]
```diff
@@ -127,6 +121,73 @@ const ExampleComponent = () => {
...
+To this end, first run the migration script from within the top-level directory of this app:
```
Comment (balegas) [inline web-wa-sqlite/README.md:140]: I would move the explanation before the code block to make the `dump.sql` less surprising :D

File: web-wa-sqlite/README.md [hunk 3]
```diff
@@ -127,6 +121,73 @@ const ExampleComponent = () => {
...
+│    npm i @prisma/client@latest                          │
+└─────────────────────────────────────────────────────────┘
+Prisma schema loaded from prisma/schema.prisma
+
+✔ Generated Prisma Client (4.8.1 | library) to ./node_modules/@prisma/client in 32ms
+
+✔ Generated Zod Prisma Types to ./prisma/generated/models in 17ms
+```
+
+The migration script updated the Electric client to incorporate the new column `other_value` on the `items` table.
```
Comment (balegas) [inline web-wa-sqlite/README.md:164]: > to incorporate the new column `other_value` on the `items` table.
Similarly to before, isn't it just modifying the type to include the new column?

**Review by balegas — COMMENTED** [review]
Left some comments

**Comment by kevin-dp** [comment]
Closing this PR to keep the examples in this repo working with the documentation on the website.
Opened another [PR](https://github.com/electric-sql/electric/pull/169) on the monorepo for this example.

**Pull request closed without merging**